scripts/policy/frameworks/intel/seen/http-headers.zeek

```diff
@@ -43,7 +31,52 @@ event http_header(c: connection, is_orig: bool, name: string, value: string)
 					}
 				}
 			break;
+			}
+		}
+	}
+
+event http_header(c: connection, is_orig: bool, name: string, value: string) &group="Intel::DOMAIN"
+	{
+	if ( is_orig )
+		{
+		switch ( name )
+			{
+			case "HOST":
```

Comment:
It seems these new handlers could be simpliefied to, rather than the switch case usage for a single entry.
```
if ( ! is_orig || or name != "HOST" )
    return;

local host = ...
Intel::seen(...);
```

As you mentioned in Slack, it's a bit unfortunate to have all these new handlers. Did you evaluate if it matters for performance much?

The ugly workaround would be to have `&group="Intel::ADDR-or-Intel::DOMAIN-or-Intel::URL`, but that'll be slightly annoying for whoever controls the groups :-(